packages/flutter_test/lib/src/matchers.dart

```diff
@@ -117,6 +117,22 @@ const Matcher isNotInCard = const _IsNotInCard();
 /// empty, and does not contain the default `Instance of ...` string.
 const Matcher hasOneLineDescription = const _HasOneLineDescription();
 
+/// Asserts that an object's toStringDeep() is a plausible multi-line
+/// description.
+///
+/// Specifically, this matcher checks that an object's
+/// `toStringDeep(prefixLineOne, prefixOtherLines)`:
+///  * Does not have leading or trailing whitespace.
```

Comment:
nit: you need a blank line before the first line with a bullet for markdown to handle this well